src/app/user-update/user-update.component.ts

```diff
@@ -0,0 +1,142 @@
+import { Component, Input, OnInit, Renderer2 } from '@angular/core';
+import { Router } from '@angular/router';
+import { FormBuilder, FormGroup, Validators } from '@angular/forms';
+import { UpdateUserService } from '../services/update-user.service';
+import { LanguageService } from '../services/language.service';
+import { ToastrService } from 'ngx-toastr';
+import { MAT_DIALOG_DATA, MatDialogRef } from '@angular/material';
+import { Inject } from '@angular/core';
+import { DashboardService } from '../services/dashboard.service';
+import { SharedDataService } from '../services/shared.data.service';
+
+@Component({
+  selector: 'app-user-update',
+  templateUrl: './user-update.component.html',
+  styleUrls: ['./user-update.component.scss']
+})
+export class UserUpdateComponent implements OnInit {
+
+  username: string;
```

Comment:
Make a model of it and then create an object.

Reply:
All of these 3 keys in the same object may not be useful given the first name and the last name key are dependent on the former.

Reply:
okay makes sense, but try doing that where ever you can like in the strides against malaria game.